homeassistant/components/postnl/sensor.py

```diff
@@ -36,7 +36,10 @@
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the PostNL sensor platform."""
-    from postnl_api import PostNL_API, UnauthorizedException
+    from postnl_api import postnl_api, PostNL_API, UnauthorizedException
```

Comment:
Could you move these imports to the top of the file? :) 
See #27284 for further information